app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_type/AddBusStopType.java

```diff
@@ -0,0 +1,52 @@
+package de.westnordost.streetcomplete.quests.bus_stop_type;
+
+import android.os.Bundle;
+
+import de.westnordost.streetcomplete.data.QuestImportance;
+import de.westnordost.streetcomplete.data.osm.OverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddBusStopType extends OverpassQuestType
+{
+	@Override
+	protected String getTagFilters()
+	{
+		return "nodes with public_transport=platform and !shelter and !informal";
```

Comment:
I see the point, and mostly agree. However, having a lot of quests have also some limits:
- You will need a lot of icon for representing them, meaning to spend some time on graphical design
- It will imply a lot of Overpass requests, which can lead sometimes to blocking from server
- If several quests are available on the same object, it will be necessary to distinguish them (or maybe not)

Here is a suggest for the form of the bus stop, which makes more sense than pictures, but still in a single quest: three icons, each one associated with a checkbox.
![bus_equipments](https://cloud.githubusercontent.com/assets/1349014/25556108/3d29549c-2cf7-11e7-8259-b1e871ba18d6.png)
This kind of layout can be made generic (GridLayout + a layout for showing an icon with a checkbox). User can give information still relatively fast. And data is fetched once for the quest. What do you think ? If it doesn't make sense, I will change for a simple yes/no layout.